app/services/onlinetesting/OnlineTestService.scala

```diff
@@ -279,51 +278,38 @@ trait OnlineTestService {
 
   private def getScheduleNamesForApplication(application: OnlineTestApplication) = {
     if (application.guaranteedInterview) {
-      gatewayConfig.onlineTestConfig.gis
+      gatewayConfig.phase1Tests.gis
     } else {
-      gatewayConfig.onlineTestConfig.standard
+      gatewayConfig.phase1Tests.standard
     }
   }
 
   private def scheduleIdByName(name: String): Int = {
-    gatewayConfig.onlineTestConfig.scheduleIds.getOrElse(name, throw new IllegalArgumentException(s"Incorrect test name: $name"))
+    gatewayConfig.phase1Tests.scheduleIds.getOrElse(name, throw new IllegalArgumentException(s"Incorrect test name: $name"))
   }
 
-  private[services] def getTimeAdjustments(application: OnlineTestApplication): Option[TimeAdjustments] = {
-    if (application.timeAdjustments.isEmpty) {
-      None
-    } else {
-      val config = gatewayConfig.verbalAndNumericalAssessment
-      Some(TimeAdjustments(
-        config.assessmentId,
-        config.verbalSectionId,
-        config.numericalSectionId,
-        getAdjustedTime(
-          config.verbalTimeInMinutesMinimum,
-          config.verbalTimeInMinutesMaximum,
-          application.timeAdjustments.get.verbalTimeAdjustmentPercentage
-        ),
-        getAdjustedTime(
-          config.numericalTimeInMinutesMinimum,
-          config.numericalTimeInMinutesMaximum,
-          application.timeAdjustments.get.numericalTimeAdjustmentPercentage
-        )
-      ))
-    }
-  }
 
   private[services] def getAdjustedTime(minimum: Int, maximum: Int, percentageToIncrease: Int) = {
     val adjustedValue = math.ceil(minimum.toDouble * (1 + percentageToIncrease / 100.0))
     math.min(adjustedValue, maximum).toInt
   }
 
   private[services] def buildInviteApplication(application: OnlineTestApplication, token: String, userId: Int, scheduleId: Int) = {
-    val scheduleCompletionUrl = s"${gatewayConfig.candidateAppUrl}/fset-fast-stream/online-tests/complete/$token"
+    val scheduleCompletionUrl = s"${gatewayConfig.candidateAppUrl}/fset-fast-stream/online-tests/phase1"
```

Comment:
I think we should be renamed to include Prefix like this scheduleCompletionUrlPrefix, scheduleCompletionBaseURL or something similar.